pages/view/_iscnId/index.vue

```diff
@@ -111,6 +112,19 @@
         :to="localeLocation({ name: 'nft-iscn-iscnId', params: { iscnId: iscnId }, query: mintQueries })"
         :text="$t('NFTPortal.button.mint')"
       />
+      <div class="flex justify-center items-center p-[4px] rounded-[12px] border-like-cyan-light border-[2px]">
+        <Button
+          v-if="!isShowMintButton"
```

Comment:
v-if=!!classId seems to be a more straight forward condition

Reply:
[4f527be](https://github.com/likecoin/app-like-co/pull/430/commits/4f527bed4c22b149766cd653e6b41851851bf034)